scripts/copy_from_upstream/copy_from_libjade.yml

```diff
@@ -0,0 +1,29 @@
+# When adding an algorithm to copy_from_libjade.yml, the boolean 
```

Comment:
What is the conceptual benefit from having a separate .YML for libjade? Why is/could this information not be integrated into our (so far one-and-only) central copy_from_upstream.yml?

Reply:
While adding libjade as an upstream it was my desire that it be possible to pull code from libjade independently of liboqs' various other upstreams (more on that below). An additional YAML file seems easier to maintain than the additional scaffolding (the already rather large) copy_from_upstream.py would require to support filtering out libjade specific information interspersed through a single YAML needed for the libjade mode. I think it also makes more sense to have separate YML files for separate modes: one for copy mode, one for libjade mode. 

Why should copy and libjade be separate modes? This PR compiles libjade *.jazz files into *.S assembly files using the Jasmin compiler instead of pulling pre-compiled *.S files from libjade releases. This approach has two advantages:

1. It allows verifying libjade proofs (e.g., correctness, constant time) in CI once libjade releases this feature.
2. We can compile the assembly files with our choice of Jasmin compiler flags, such as those enabling emitting DWARF2 info.

However, the Jasmin compiler isn't widely packaged (only available in Nix and OPAM). Also, compiling the assembly files locally takes considerable time. Requiring all liboqs developers to install a not-easy-to-install package and compile libjade even to update non-libjade code would hinder development.

Reply:
Thanks for the explanation. LGTM.